compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -2688,6 +2717,21 @@ private ParserRuleContext getNextRuleForKeywords(ParserRuleContext currentCtx, i
                     endContext(); // end let-clause-let-var-decl
                 }
                 return ParserRuleContext.EXPRESSION;
+            case ORDER_KEYWORD:
+                parentCtx = getParentContext();
+                if (parentCtx == ParserRuleContext.LET_CLAUSE_LET_VAR_DECL) {
```

Comment:
I think we shouldn't need to add it here, since an order-keyword cannot exist inside a let-var-decl.